packages/fxa-auth-server/lib/senders/emails/layouts/subscription/en.ftl

```diff
@@ -1,4 +1,5 @@
 subplat-automated-email = This is an automated email; if you received it in error, no action is required.
+subplat-privacy-notice = Privacy notice
 subplat-privacy-plaintext = Privacy notice:
```

Comment:
I wonder what the implications are around changing `subplat-privacy-plaintext` to equal `{ subplat-privacy-notice }:` (might be good to run by @peiying2 just so we know for sure), but leaving them separate as you did is probably safest since that ID was already created.

Reply:
To avoid any implications, I have left the separate.